### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by roshan-bajgain**
[Video link](https://www.loom.com/share/e3756d0a05694a7aaa39135d9602522f)
Create new repo and add HTML, CSS, and js files and use a linter.
first, make a mobile page like the first page and the about page. 
Then implement those in the desktop version and using js first make a mobile menu and create the dynamic parts by javascript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
-# Conference-App
\ No newline at end of file
+# capstone project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# A Microverse capstone project
+
```
Comment (henatan99) [inline README.md]: - [x] I think it is better to use a more descriptive title here than just "A Microverse capstone project". 
- [x] Kindly write a brief description of the project in the about section. Currently, users who visit your repo will not have a good picture of what the project is about.

File: README.md [hunk 2]
```diff
@@ -1 +1,52 @@
...
+## Built With
+
+- HTML
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://roshan-bajgain.github.io/Conference-App/)
```
Comment (henatan99) [inline README.md:17]: - [x] The live link displays just a copy of your readme content. It should be the app itself with Home and About pages with the contents and design shown in the original design reference. :pray: 

Screenshot from 2021-09-15 15-22-26

File: about.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        <nav class="navbar navbar-expand-md navbar-light">
+            <div class="container-fluid">
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                    <ul class="navbar-nav">
+                        <li class="nav-item "><a href="index.html"><img id="logo" src="img/icons/logo.png"
```
Comment (henatan99) [inline about.html:35]: - [x] There is no `index.html` file in your project. As a result, this link redirects to nowhere but to a blank browser window with an error message. 
image

- [x] Kindly add the Home page (index.html) as the project requires the web app should have two pages, a Home page and About page.

File: about.css [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+
+@media (min-width: 768px) {
+  #greeting {
+    font-size: 45px;
+    margin-left: 0;
+  }
+}
+
+#topic {
+  color: orangered;
```
Comment (henatan99) [inline about.css:149]: - [x] I think the right color here should be #f43d28. You can use color-picker google extension to get the colors from the design reference. 

image
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+            </ul>
+        </nav>
+        <nav class="navbar navbar-expand-md navbar-light">
+            <div class="container-fluid">
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarNavDropdown">
```
Comment (henatan99) [inline about.html:33]: - [x] In the original design, the bottom bar contains a logo to the left.
- [x] In the original design, the bottom bar contains a list of page links aligned to the right.
- [x] Kindly align the logo to the left and the links to the right according to the design requirement. :pray: 

image

image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+                    you will learn libraies and frameworks, such as SASS, bootstrap, and React to help you build a
+                    website in a even faster way.
+                    In addtion, you will learn back end framworks Ruby on Rails to build a website with database which
+                    can store and manage user
+                    and activity data. After you can handle both front end and back end project, you are ready to apply
+                    these knowledge to build web
+                    application that solve real-world problem in all kinds of fields. The career center is there to help
+                    you connect with companies that
+                    has plenty of in-demands request from their customer. You job and opportunity are already there!
+                </article>
```
Comment (henatan99) [inline about.html]: - [x] This is a paragraph and semantically it is supposed to have a `p` tag. An article is kind of a wrapper that can have other `hx`, `p` 'img`, etc elements. Kindly make a change here. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @roshan-bajgain , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] I cannot see any video recording link about the project either in your PR summary or the readme file. Kindly record video documentation about the project and provide the link either in your PR summary or your readme. :pray: 
- [x] You must add the Home page (index.html). The project requires a web app with two pages, the Home page, and the About page. Carefully follow the [project's requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) and the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to build your Home page. :pray: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 7]
```diff
@@ -1 +1,56 @@
...
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Conference page
+  - This is my Conference page. With the proper flow of git and Linter I make this page.
+  Firstly I make mobile website using basic HTML ans CSS. After that I worked on bigger screen like desktop and with 
+  Adjusticable width I work on desktop. After completing the desktop I work on js for making the mobile menu  appear.
+  After I make the website dymanic using js.
+
+## Built With
+
```
Comment (george-shammar) [inline README.md:11]: - [ ] In your video, you did not present any project. You simply talked about what you did. Please record another video and showcase your project by talking through what you did.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @roshan-bajgain , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @roshan-bajgain 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:
- The live link is working properly:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewers. 
 
 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**